### PR TITLE
Dashboard

### PR DESCRIPTION
**Pull request by CptTony**
Implementation of account upgrade offer

### CODE REVIEW

File: src/containers/Dashboard/index.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ import {
   Label,
 } from 'recharts';
 
+import metricsService from '../../services/metrics.service';
```
Comment (simonebelotti75) [inline src/containers/Dashboard/index.js:12]: Use absolute import

File: src/containers/Dashboard/index.js [hunk 2]
```diff
@@ -154,36 +202,48 @@ function DashboardPage(props) {
...
-                    <div className="row mt-3">
-                      <div className="col">
-                        <a className="btn btn-primary submit" onClick={handleUpgradeClick}>Upgrade to Basic Account</a>
-                      </div>
-                    </div>
-                    <div className="row mt-1 mb-3">
-                      <div className="col">
-                        It&apos;s FREE. Courtesy of Linode.com
+              {
+                (metrics.membership_type_name === 'demo') ?
```
Comment (simonebelotti75) [inline src/containers/Dashboard/index.js:215]: Instead of ?, please use &&

File: src/containers/Dashboard/index.js [hunk 3]
```diff
@@ -154,36 +202,48 @@ function DashboardPage(props) {
                     {metrics.membership_type}
                   </div>
                   <div className="col text-right">
-                    <a className="btn btn-primary submit mr-5" onClick={handleUpgradeClick}>Upgrade Now</a>
+                    {
+                      (metrics.membership_type_name === 'demo') ?
```
Comment (simonebelotti75) [inline src/containers/Dashboard/index.js:206]: Instead of ?, please use &&

**Review by simonebelotti75 — CHANGES_REQUESTED** [review]

File: src/containers/Dashboard/index.js (hunk 1 above)
Comment (CptTony) [reply src/containers/Dashboard/index.js]: Fixed

File: src/containers/Dashboard/index.js (hunk 2 above)
Comment (CptTony) [reply src/containers/Dashboard/index.js]: Fixed

File: src/containers/Dashboard/index.js (hunk 3 above)
Comment (CptTony) [reply src/containers/Dashboard/index.js]: Fixed

**Pull request merged**